plugins.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: default
+title: Plugins
+headline: Pico Plugins
+description: |
+  Want to extend the functionality of Pico?  Below is a collection of our community-developed plugins.<br>
+  Would you like to create your own plugin? Learn how in [our documentation](/docs/#plugins)!
+nav: 5
+
+portfolio:
+  layout: card
+  categories:
+    admin: Administration
+    developer: Developer-Focused
+    theming: Theming Extensions
+  defaultThumbnail: plugins/images/thumbnails/plugin.png
+  defaultImage: themes/images/fullsize/default_theme.png
+
+---
+
+{% include portfolio.html portfolio=site.plugins %}
+
+---
+
+## More Plugins & How to Contribute Your Own
+
+The plugins shown here are only a smaller selection of those available.  You can find more themes on [our Wiki][].
+
+You can even [contribute][] your own plugin to the community and we'll feature it here.  While you can freely add your plugin to [our Wiki][], if you'd also like to be included on this page, you can follow the simple instructions in our [Submission Guidelines][].
```

Comment:
Yes, the image is a work in progress.  I started to write that, then I realized that essentially *everything* was work in progress, so it felt redundant to say.

Huh, I didn't realize (or forgot) that you made it so we can set the layout on a per-item basis.  I'll do that.

Personally, I think it belongs at the end of the list.  The concept being that, by contributing, you'd make the list longer.  Also, it didn't look good at the beginning. :laughing:  It started at the beginning of the Plugins page because nothing came alphabetically before "Contribute".  So the `zzz_` was a quick way to make it always be at the end of the list.